test/junit/scala/tools/testing/AllocationTest.scala

```diff
@@ -152,13 +152,5 @@ trait AllocationTest {
 case class AllocationExecution(executionCount: Int = 1000, warmupCount: Int = 1000)
 
 case class AllocationInfo[T](result: T, allocations: Array[Long]) {
-  def min: Long = {
-    var min = allocations(0)
-    var i = 1
-    while (i < allocations.length) {
-      min = Math.min(min, allocations(i))
-      i += i
-    }
-    min
-  }
+  def min = allocations.min
 }
```

Comment:
I wondered if it was written this way in order to avoid allocating, but we have the same change on the 2.13 branch: d79f001a3b1